html/workout.html

```diff
@@ -7,28 +7,171 @@
                     <li><a href="workout.html" class="current-page">Workout</a></li>
                     <li><a href="contact.html">Contact</a></li>
                 </ul>
             </nav>
         </div>
+
     </header>
     <!--NAVIGATION BAR ENDS-->
+
+    
+    <div class="main-container">
+        
+        <main>
+
+            <div id="form1">
+
+                <form>
+                    <h4 style="color: rgba(255,255,255,.70);"> What's Your Name: </h4>
+                    <div class="row">
+                        <div class="col">
+                            <input type="text" class="form-control" placeholder="First name" id="fname">
+                        </div>
+                        <div class="col">
+                            <input type="text" class="form-control" placeholder="Last name" id="lname">
+                        </div>
+                    </div>
+                </form>
+
+                
+
+                <button id="enterNameButton" class="btn btn-primary submitButton"> Confirm </button>
+
+
+                
+                <p class="animated-p" id="welcomeMessage" style="display: none;">Hey </p>
+
+                <p class="animated-p" id="welcomeMessage2" style="display: none;">Let's Get Started...</p>
+
+
+                <p class="animated-p" id="message3" style="display: none;">Do you want to build muscle or lose weight?</p>
+
+
+                <div id="buildmusOrLoseWeight" style="display: none;">
+                    <div class="custom-control custom-radio custom-control-inline">
+                        <input type="radio" id="buildM" name="Radio1" class="custom-control-input">
+                        <label class="custom-control-label" for="buildM"><p>Build Muscle</p></label>
+                    </div>
+                    <div class="custom-control custom-radio custom-control-inline">
+                        <input type="radio" id="loseW" name="Radio1" class="custom-control-input">
+                        <label class="custom-control-label" for="loseW"><p>Lose Weight</p></label>
+                    </div>
+                </div>
+
+
+                <button style="display: none;" id="cont1" class="btn btn-primary submitButton"> Continue </button>
+
+
+                <p class="animated-p" id="message4" style="display: none;">Great, let's </p>
+
+
+                
+                <form style="display: none;">
+                    <p>Do You Have Access To a Gym?</p>
+                    <input type="radio" id="yestogym" name="accessToGym" value="Yes">
+                    <label for="yestogym">Yes</label><br>
+                    <input type="radio" id="notogym" name="accessToGym" value="No">
+                    <label for="notogym">No</label><br>
+                </form>
+
+
+                
+    
+            </div>
+    
+
+        </main>
+        
+        
+    </div>
+
+    <script>
+
+        var firstName = "";
+        var lastName = "";
+

```

Comment:
Add this to keep the background color the gray we follow:
```suggestion

document.body.style.backgroundColor = "#606060";
```